src/test/obj_ulog_size/obj_ulog_size.c

```diff
@@ -41,17 +41,22 @@
 #include "unittest.h"
 
 /*
- * lane.h -- needed for LANE_REDO_EXTERNAL_SIZE
+ * lane.h -- needed for LANE_REDO_EXTERNAL_SIZE and LANE_UNDO_SIZE
  */
 #include "lane.h"
 
 /* TX_INTENT_LOG_ENTRY_OVERHEAD -- sizeof(struct ulog_entry_val) */
-#define TX_INTENT_LOG_ENTRY_OVERHEAD (0b01ULL << 4) /* 16 bytes */
-#define TX_SNAPSHOT_ALIGNMENT CACHELINE_SIZE
+#define TX_INTENT_LOG_ENTRY_OVERHEAD 16ULL
+
+/* TX_SNAPSHOT_ALIGNMENT -- CACHELINE_SIZE */
+#define TX_SNAPSHOT_ALIGNMENT 64ULL
 #define TX_SNAPSHOT_LOG_BUFFER_OVERHEAD 64ULL
+
 /* TX_INTENT_LOG_ENTRY_OVERHEAD -- sizeof(struct ulog_entry_buf) */
-#define TX_SNAPSHOT_LOG_ENTRY_OVERHEAD (0b11ULL << 3) /* 24 */
-#define TX_INTENT_ALIGNMENT CACHELINE_SIZE
+#define TX_SNAPSHOT_LOG_ENTRY_OVERHEAD 24ULL
+
+/* TX_INTENT_ALIGNMENT -- CACHELINE_SIZE */
+#define TX_INTENT_ALIGNMENT 64ULL
 #define TX_INTENT_LOG_BUFFER_OVERHEAD 64ULL
```

Comment:
The comments describing where these numbers come from do not solve the issue for me.
- We define defines which already exist somewhere else.